### PR TITLE
Enable show_2dspec script to run on older datamodels

### PR DESCRIPTION
**Pull request by profxj**
As titled.

I have not included a test, but i have run this on a v1.0.0 spec2d file successfully.

Unit tests passed except for a few which need the new Cooked tarball
that @rcooke-ast  is currently uploading.

### CODE REVIEW

**Comment by profxj** [comment]
Passing on all Kast:

--- PYPEIT DEVELOPMENT SUITE PASSED 13/13 TESTS (Masters ignored) ---

**Comment by profxj** [comment]
Running pypeit on gemini_gnirs/10_LB_SXD (ignores masters)  --- PASSED

**Comment by profxj** [comment]
Still passing `shane_kast`

--- PYPEIT DEVELOPMENT SUITE PASSED 13/13 TESTS (Masters ignored) ---

File: pypeit/scripts/chk_edges.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def parser(options=None):
                         help='Channel name for image in Ginga')
     parser.add_argument('--mpl', default=False, action='store_true',
                         help='Use a matplotlib window instead of ginga to show the trace')
+    parser.add_argument('--try_old', default=False, action='store_true',
+                        help='Attempt to load old datamodel versions.  A crash may ensew..')
```
Comment (jhennawi) [inline pypeit/scripts/chk_edges.py:23]: ensue

File: pypeit/scripts/chk_flats.py [hunk 2]
```diff
@@ -13,10 +13,12 @@ def parser(options=None):
     parser.add_argument("--type", type=str, default='all', help="Which flats to display. Must be one of: pixel, illum, all")
     parser.add_argument('master_file', type=str,
                         help='PypeIt MasterFlat file [e.g. MasterFlat_A_1_01.fits]')
+    parser.add_argument('--try_old', default=False, action='store_true',
+                        help='Attempt to load old datamodel versions.  A crash may ensew..')
```
Comment (jhennawi) [inline pypeit/scripts/chk_flats.py:17]: ensue

File: pypeit/scripts/chk_edges.py (hunk 1 above)
Comment (profxj) [reply pypeit/scripts/chk_edges.py:23]: fixed (in both)

**Comment by profxj** [comment]
================ 234 passed, 753 warnings in 433.30s (0:07:13) ================

**Pull request merged**